Makefile

```diff
@@ -80,7 +80,7 @@ setup: ## Download controller-gen and  locally if necessary.
 run: manifests generate ## Run a controller from your host.
 	go run ./main.go
 
-build:
+build: update-chart
```

Comment:
@paynejacob I added this here because I found out `make build` won't actually update the CRDs and roles

Reply:
this should be manual by the developer